miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/BulkSampleEditIT.java

```diff
@@ -285,6 +285,90 @@ public void testEditSaveEmptySaveTissue() throws Exception {
     assertAllForTissue(empty, getIdForRow(newTable, 0), false);
   }
 
+  @Test
+  public void testDistributeTissue() throws Exception {
```

Comment:
This is really here to test the Service and DAO rather than the UI. Would be simpler and faster to use unit tests instead - I think a test for `DefaultSampleService#applyChanges` and another for `HibernateSampleDao#save` could do the job

Reply:
I was actually trying to test the UI effects (that the box field would be cleared and that the Distribution Date and Distribution Recipient would only be editable after Distributed is set to Sent Out) but I didn't make that clear in my test! Will update the test to confirm it does what I want it to, and will add unit tests as well.

Reply:
Those UI effects could be tested more simply by loading the edit page for a sample already in a box and changing the distribution properties. No saves would be necessary. There's `HandsOnTable#isWritable` to check if fields are writable or read-only too